static/js/tools/choropleth/choropleth.ts

```diff
@@ -97,23 +113,25 @@ function generateGeoMap(geojson, plt_info) {
 function handleMapHover(geo) {
```

Comment:
I stronger suggest to make a Map React component and move these logic (together with the ones below like mouseLeave, etc) into it.

Ideally, this file should only contain a few lines as the javascript entrance file.